src/nectarchain/calibration/container/waveforms.py

```diff
@@ -136,138 +157,196 @@ def check_events(self):
         has_changed = True
         previous_trigger = None
         n_events = 0
-        for i,event in enumerate(self.__reader):
-            if previous_trigger is not None :
-                has_changed = (previous_trigger.value != event.trigger.event_type.value)
+        for i, event in enumerate(self.__reader):
+            if previous_trigger is not None:
+                has_changed = previous_trigger.value != event.trigger.event_type.value
             previous_trigger = event.trigger.event_type
-            if has_changed : 
-                log.info(f"event {i} is {previous_trigger} : {previous_trigger.value} trigger type")
-            n_events+=1
+            if has_changed:
+                log.info(
+                    f"event {i} is {previous_trigger} : {previous_trigger.value} "
+                    f"trigger type"
+                )
+            n_events += 1
         return n_events
 
-
-
-    def load_wfs(self,compute_trigger_patern = False):
-        """mathod to extract waveforms data from the EventSource 
+    def load_wfs(self, start, end, compute_trigger_patern=False):
```

Comment:
A similar comment here, I think `start` and `end` should have default values such that the all events should be loaded.